src/messageschannel.cpp

```diff
@@ -30,14 +30,17 @@
 #include <connection.h>
 #include <room.h>
 #include <user.h>
+#include <csapi/typing.h>
+#include <events/typingevent.h>
 
 MatrixMessagesChannel::MatrixMessagesChannel(MatrixConnection *connection, QMatrixClient::Room *room, Tp::BaseChannel *baseChannel)
     : Tp::BaseChannelTextType(baseChannel),
       m_connection(connection),
       m_room(room),
       m_targetHandle(baseChannel->targetHandle()),
       m_targetHandleType(baseChannel->targetHandleType()),
-      m_targetId(baseChannel->targetID())
+      m_targetId(baseChannel->targetID()),
+      m_localTypingTimer(0)
```

Comment:
Please use 'in class definition' initialization instead. (I know I have (or had?) such one in Morse but let's avoid it in new code.)